ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -1089,4 +1089,80 @@ spec:
   tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_2.3.17.8
   contributors: rachelelysia
 ---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Prevent enabling lock screen camera' is set to 'Enabled' (Automated)
+  platforms: win10
+  platform: windows
+  description: |
+    Disables the lock screen camera toggle switch in PC Settings and prevents a camera from being invoked on the lock screen.
+    The recommended state for this setting is: Enabled.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to Enabled:
+    'Computer Configuration\Policies\Administrative Templates\Control Panel\Personalization\Prevent enabling lock screen camera'
+  query: |
+    SELECT 1 FROM registry where (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows\\Personalization\NoLockScreenCamera' AND data == 1);
```

Comment:
one small cosmetic thing, the registry path should have \\ for all the separators (the last one just has \)

Reply:
It was initially `:` but it shows up as a single `\`, lets chat on zoom